app/models/provider.rb

```diff
@@ -63,4 +64,10 @@ def address_info
       .attributes_before_type_cast
       .slice('address1', 'address2', 'address3', 'address4', 'postcode', 'region_code')
   end
+
+  def update_changed_at(timestamp: Time.now.utc)
+    # We don't want `updated_at` to change, only `changed_at`. Otherwise what
+    # would be the point of separating the two?
+    update_columns changed_at: timestamp
```

Comment:
` update_columns changed_at:  Time.now.utc` instead?

The need for the parameter? Other then for test, maybe timecop?

Reply:
Not having a parameter her makes me itchy. It's trivial to do and it means no further change needed if we need it in the future. I don't think it gets in the way here, do you?

Reply:
I'm not fussed, just stating.